pkgs/development/python-modules/bitbox02/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, buildPythonPackage
+, hidapi
+, noiseprotocol
+, protobuf
+, ecdsa
+, semver
+, typing-extensions
+, base58
+, lib
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "bitbox02";
+  version = "2.0.3";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1mcbg7bvlsii1kj2mm5686z01pwcc36qbqlhz4a8syjrzjm6pl2k";
+  };
+
+  doCheck = false;
```

Comment:
Please add a comment why you skip tests or fix the tests, so they don't need to be skipped.